rusk-recovery/src/bin/presets.rs

```diff
@@ -0,0 +1,99 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use rusk_recovery_tools::state::Snapshot;
+
+pub fn localnet() -> Snapshot {
+    let toml = include_str!("../../configs/localnet.toml");
```

Comment:
Doesn't this ensure `rusk-recovery` can only be called in this directory? We might want to include the file directly.

Reply:
Disregard. I assumed that this included the string and not the file at compile time